contrib/nydusd-http-server/README.md

```diff
@@ -0,0 +1,105 @@
+# Http server for nydusd
+A http server based on Rocket to serve static file for nydusd.
+
+
+## APIs supported
+support the following APIs:
+```bash
+HEAD /$namespace/$repo/blobs/sha256:xxx #get blob size
+GET /$namespace/$repo/blobs/sha256:xxx #get blob range data
+```
+### Existing Layers
+```
+HEAD /v2/<name>/blobs/<digest>
+```
+On Success: OK
+```
+200 OK
+Content-Length: <length of blob>
+Docker-Content-Digest: <digest>
+
+```
+### Fetch Blob
+```
+GET /v2/<name>/blobs/<digest>
+Host: <registry host>
+Authorization: <scheme> <token>
```

Comment:
Does the server supports `Authorization`? If not, the header definition is unnecessary.